src/content/resources/index.mdx

```diff
@@ -19,20 +20,39 @@ We have resources to help you navigate the OASIS+ contract; as part of this, we
+| [Symphony Task Order System - How to use the portal](https://industrysupport.apexlogic.com/support/solutions/articles/35000224610-how-to-use-the-oasis-symphony-portal) | Provides instructions for getting started with Symphony. |
+| [Symphony Task Order System - How to build an offer](https://industrysupport.apexlogic.com/support/solutions/articles/35000222331-how-to-build-your-offer-for-oasis-) | Provides instructions for building an offer in Symphony. |
+| [Frequently Asked Questions](https://industrysupport.apexlogic.com/support/solutions/articles/35000223291-oasis-faq) | Frequently asked questions about Symphony. |
 | Training - proposal submission (coming soon) | Learn how to submit your proposal in Symphony. |
-| Sellers training - task order management (coming soon) | Learn about creating and managing your task orders in Symphony. |
+| Sellers training - task order management   (coming soon) | Learn about creating and managing your task orders in Symphony. |
 
 ### Buyers
 | Resource | Description |
 | --------------------------| --------------------------|
-| Buyers training - task order management (coming soon) | Learn about creating and managing your task orders in Symphony. |
-| [Steps to Performance Based Acquisition](https://buy.gsa.gov/spba) | Website presenting the PBA process in easy steps, complete with samples & examples, templates, and other resources to make PBA more collaborative, performance-oriented, and team-focused.|
+| [OASIS+ Fact Sheet (PDF)](https://github.com/GSA/oasis-plus/blob/vickimcfadden-patch-1/public/files/OASIS%2B%20Fact%20Sheet.pdf) | A high-level overview of the OASIS+ contract scope, features and benefits, ordering process, and related eTools. |
+| [Symphony - The task order tool that works for you](https://www.youtube.com/watch?v=cQ4zVfA8MAY) | Video tutorial that gives an overview of Symphony.|
+| [Symphony Task Order System - How to register](https://gettingstarted.apexlogic.com/support/solutions/articles/35000177795-registering-as-a-government-user)| Provides instructions for how to register with Symphony. |
+| Getting started - Frequently Asked Questions | Frequently asked questions about Symphony. |
+| [How to Use Symphony Task Order Management for OASIS+ (video)](https://www.youtube.com/watch?v=LXkqNV88FJI)| Learn how to submit proposals and capture information in Symphony, the database of record for all stakeholders. 
+
+Note: this video references OASIS, but the information remains relevant to OASIS+.|
+| [Symphony Task Order Management for OASIS+ Customers (PDF)](https://www.gsa.gov/system/files/Final_-_Symphony_Customer_Rollout_Presentation_%282022-03-17%29_.pdf) | Learn more about Symphony’s benefits, role in the ordering process, and additional resources related to training and support.
+
```

Comment:
In the preview, this section isn't showing as part of the table (and I think it's intended to?) 
<img width="1016" alt="Screenshot 2024-05-31 at 2 05 33 PM" src="https://github.com/GSA/oasis-plus/assets/18366192/004e9a6b-293f-46ea-bfc4-7cfcaf3d7726">

Reply:
resolved